src/main/java/org/takes/rs/RsWithCookie.java

```diff
@@ -52,6 +52,10 @@
 public final class RsWithCookie extends RsWrap {
 
     /**
+     * Matcher expression for non printable chars.
+     */
+    private static final String INVALID_CHARS = "[^\\p{Print}]";
```

Comment:
@pinaf I'm not sure, we have to find a proper RFC and mention it here, in the javadoc. Also, I think it should be `Pattern` instead of `String`